src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -133,9 +133,10 @@ var inputValueTracking = {
       return false;
     }
     var tracker = getTracker(subject);
+    const isNode = (subject: any).nodeType === ELEMENT_NODE;
 
     if (!tracker) {
-      if (typeof (subject: any).tag === 'number') {
+      if (isNode || typeof (subject: any).tag === 'number') {
```

Comment:
So don't think `subject` can actually be a `Fiber`, unless that is what is being passed in in the ChangeEventPlugin, but i'm not sure how to confirm it. If so this should be easier to simply

I would split this method into `updateValueIfChangedFiber` or something and then when Stack is retired, delete the old method, rename and move on.

Is there something that needs to be done to run tests against either Stack or Fiber?